### PR TITLE
adjusted movedeffs

### PR DESCRIPTION
**Pull request by drivver44**
### Work done
adjusted all freely available movedeffs (jugg is not included but is planned 7x7)
screenshots will be coming later
<!-- If relevant
#### Addresses Issue(s)
gdt thread

screennshots will come later

### CODE REVIEW

**Comment by drivver44** [comment]
minor note
jugg would be moved from a 5x5 movedeff to a 7x7

File: gamedata/movedefs.lua [hunk 1]
```diff
@@ -57,8 +57,8 @@ local moveDatas = {
 	ABOT2 = {
 		crushstrength = 50,
 		depthmod = 0,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:61]: Shouldn't this be called ABOT3 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 2]
```diff
@@ -80,8 +80,8 @@ local moveDatas = {
 	ATANK3 = {
 		crushstrength = 30,
 		depthmod = 0,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:84]: Shouldn't this be called ATANK4 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 3]
```diff
@@ -191,8 +191,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 25,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:195]: Shouldn't this be called HOVER3 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 4]
```diff
@@ -203,8 +203,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 25,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:207]: Shouldn't this be called HOVER4 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 5]
```diff
@@ -244,8 +244,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 252,
-		footprintx = 4,
-		footprintz = 4,
+		footprintx = 5,
+		footprintz = 5,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:248]: Shouldn't this be called HOVER5 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 6]
```diff
@@ -293,8 +293,8 @@ local moveDatas = {
 	-- cormlv armmflash corgator legmrv armmlv leghades leghelops armfav corfav armconsul armlatnk cortorch legmrrv
 	TANK2 = {
 		crushstrength = 18,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:297]: Shouldn't this be called TANK3 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 7]
```diff
@@ -339,8 +339,8 @@ local moveDatas = {
 	--corprinter corvac corvacct correap corftiger armbull legsco corvoc armmerl
 	MTANK3 = {
 		crushstrength = 250,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:343]: Shouldn't this be called MTANK4 now? It seemed the name was intended to match the foorptinr size,

File: gamedata/movedefs.lua [hunk 8]
```diff
@@ -502,8 +502,8 @@ local moveDatas = {
 	-- armvang corkarg corthermite
 	HTBOT4 = {
 		crushstrength = 252,
-		footprintx = 4,
-		footprintz = 4,
+		footprintx = 5,
+		footprintz = 5,
```
Comment (marcushutchings) [inline gamedata/movedefs.lua:506]: Shouldn't this be called HTBOT5 now? It seemed the name was intended to match the foorptinr size,

**Review by marcushutchings — COMMENTED** [review]
Perhaps we need a new naming scheme for movetypes?

ABOT2 -> AmphibiousBotSmall

Or something like that.

File: gamedata/movedefs.lua (hunk 1 above)
Comment (drivver44) [reply gamedata/movedefs.lua:61]: i can do that at a later time
im going to soft resolve all of the naming issues later before a merge occurs

**Comment by drivver44** [comment]
> Perhaps we need a new naming scheme for movetypes?
> 
> ABOT2 -> AmphibiousBotSmall
> 
> Or something like that.

honestly the whole naming scheme of movedeffs should be reworked
but i think that would be best done as a seperate PR to this PR for ease of looking at the actual changes for a better more spread out history

**Comment by drivver44** [comment]
mass list of all changes for documentation purposes
```
please note that this is using a single unit as a refference for the entire movedeff change
for all units affected do look at movedeffs.lua for the full list of unitdeff names affected
```

commander movedeff changes `COMMANDERBOT`
proposed
image
current
image


pawn `BOT3`
proposed
image
current 
image

hound(fido) `BOT4`
proposed 
image
current
image
side note i think hound should go to bot3 but its in bot4 currently
just an optimization thats possible and best followed through

jethro `abot2`
proposed
image
current
image

poison arrow `atank3`
proposed 
image
current
image
note needs split here 
corgarp uses this movedeff and does not really belong on this movedeff with this specificed spacing

sweeper `hover2`
proposed
image
current
image

halberd `hover3`
proposed
image
current
image
needs slight split and merge here 
some of the hover tanks do not fit this movedeff now and very much fit into `hover2`

lunkhead `hhover4`
proposed
image
current
image

platypus `hover5`
proposed
image
current
image

blitz `tank 2`
proposed 
image
current
image

printer `mtank3`
proposed 
image
current
image

kargeneth
proposed
image

current
image

**Comment by marcushutchings** [comment]
All my comments are mearly suggestions. If there's a bigger plan in motion then that's fine. But be careful of increasing the footprint if it is purely to spread out units - it will also impact the gaps between builds that units can walk through: i.e. they can get trapped more easily.

**Comment by 6AKU66** [comment]
What is the goals of those adjustment btw?

**Comment by 6AKU66** [comment]
Personally i'm for adjustments when it's will help avoid clipping of units to each other.

**Comment by drivver44** [comment]
In short it’s to allow units to not stack up in ways that brick the units
from not firing

**Comment by 6AKU66** [comment]
Hm... I don't see that it's a good idea adjusting movedeffs for fixing this problem. And i think it's gonna cause problems like:
1. Units will get more easily stuck not only in buildings, but in trees, rocks, unit's wrecks and in shallow map's gaps;
2. When units gonna collide it's gonna looks strange when unit's model pushing another unit's model telekinetically;
3. Players gonna have less control over how compact they want them's army to be.

**Comment by drivver44** [comment]
due to changes in how the unit seperation works currently in engine 
this pr is DOA 
the change from movedeffs to unit deffs is the key change
i dont intend to add support on the side of movedeffs as that is likely not needded
and reduces the tetris effect

**Pull request closed without merging**